src/als/ui/dialogs.py

```diff
@@ -121,7 +131,19 @@ def browse_work(self):
         if work_folder_path:
             self._ui.ln_work_folder_path.setText(work_folder_path)
 
-        self._show_missing_folders()
+        self._validate_all_pathes()
+
+    @pyqtSlot(name="on_btn_dark_scan_clicked")
+    @log
+    def browse_dark(self):
+        """Opens a folder dialog to choose dark file"""
+        dark_file_path = QFileDialog.getOpenFileName(self,
+                                                     _("Select dark file"),
+                                                     self._ui.ln_master_dark_path.text())
+        if dark_file_path[0]:
+            self._ui.ln_master_dark_path.setText(dark_file_path[0])
+
+        self._validate_all_pathes()
```

Comment:
This call to self._validate_all_pathes() on line 146 is useless. Sorry, I misguided you on this one

Reply:
Done ;)